streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -774,48 +771,6 @@ public void shouldNotCommitNonRunningNonCorruptedTasks() {
         verify(consumer);
     }
 
-    @Test
-    public void shouldCleanAndReviveCorruptedStandbyTasksBeforeCommittingNonCorruptedTasks() {
```

Comment:
This also doesn't apply any more -- it was a test I had added in response to [KAFKA-10144](https://issues.apache.org/jira/browse/KAFKA-10144), a bug caused by the commit in `handleCorruption`